### ISSUE TITLE
[Feature Request] Azure Active Directory

### ISSUE THREAD

**Issue by cocowalla** [nraboy/ng2-cordova-oauth#17]
Would be great to add support for Azure Active Directory, as the Angular v1 version [has](https://github.com/nraboy/ng-cordova-oauth/blob/a4a83687a3c5f517f66579a47ba06436f6ab612b/src/oauth.azuread.js).

**Comment by pavankumarkatakam** [comment]
Yeah, Azure active directory is important to me for now because of I want to work with Dynamics API.
